etl/steps/data/garden/oecd/2024-04-10/income_distribution_database.py

```diff
@@ -0,0 +1,219 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from typing import List
+
+import owid.catalog.processing as pr
+from owid.catalog import Table
+from structlog import get_logger
+from tabulate import tabulate
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+# Initialize logger.
+log = get_logger()
+
+# Define new names for categories
+INDICATOR_NAMES = {
+    "Gini (disposable income)": "gini_disposable",
+    "Gini (gross income)": "gini_gross",
+    "Gini (market income)": "gini_market",
+    "P50/P10 disposable income decile ratio": "p50_p10_ratio_disposable",
+    "P90/P10 disposable income decile ratio": "p90_p10_ratio_disposable",
+    "P90/P50 disposable income decile ratio": "p90_p50_ratio_disposable",
+    "Palma ratio (disposable income)": "palma_ratio_disposable",
+    "Quintile share ratio (disposable income)": "s80_s20_ratio_disposable",
+    "Poverty rate based on disposable income": "headcount_ratio_disposable",
+    "Poverty rate based on market income": "headcount_ratio_market",
+}
+
+POVERTY_LINES = {
+    "Not applicable": "Not applicable",
+    "50% of the national\xa0median disposable income": "50_median",
```

Comment:
Are these names expected to be human readable, e.g. "Not applicable" or machine readable, e.g. "50_median"?

Reply:
I use it for them to be machine readable and not applicable is not used, but I changed it now to not_applicable for completeness